.github/workflows/ci.yml

```
@@ -7,6 +7,8 @@ on:
       - main
     tags:
       - v*
+    schedule:
+      - cron: "0 0 * * 1-5"
   pull_request:
 
```

(optional) Should we also have a `workflow_dispatch` here, so we can manually trigger if needed? eg similar to:
https://github.com/heroku/cnb-builder-images/blob/0a8816f18165c7f14e8841b3b931bb67a9d351a5/.github/workflows/build-test-publish.yml#L9

```suggestion
  workflow_dispatch:

```